code/src/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Artist } from './Artist'
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.